### ISSUE TITLE
Linux CLI help shows wrong description

### ISSUE THREAD

**Issue by fschwarzer** [Azure/iothub-explorer#5]
Exerpt of ```$ iothub-explorer --help```:
```
    query-twin <sql-query>                                                         Gets the twin of a device
    query-job [job-type] [job-status]                                              Gets the twin of a device
    device-method <device-id> <method-name> [method-payload] [timeout-in-seconds]  Gets the twin of a device
```

See description ...

**Comment by pierreca** [comment]
Fixed with https://github.com/Azure/iothub-explorer/commit/137cd677e95825c8f93105991c0a33ce4ce1b8fd

We will close the issue when the new package is released.

**Comment by pierreca** [comment]
The fix has been released with iothub-explorer@1.1.4 on npm.
